controllers/apiArticlesController.js

```diff
@@ -0,0 +1,9 @@
+const { fetchArticles } = require("../models/apiArticlesModel");
+
+const getArticles = (request, response) => {
+  return fetchArticles().then((articles) => {
+    response.status(200).send({ articles });
+  });
```

Comment:
Very nice, it might be wise to include a `.catch` here just in case the query encounters any errors so they can be passed back to _app.js_

Reply:
done